test/smoke-fort/milestone-3-babel/doit.sh

```diff
@@ -2,11 +2,16 @@
 save_status() { tval=$? && (($tval!=0)) && rval=$tval; }; rval=0
 echo "ZZZZZZZZZZ BabelStream"   # marker for extract-tests
 set -x
-./milestone-3-babel -n 20               # 805.3 MB
+PRECMD=""
+if [ -f $AOMP/bin/gpurun  ]; then
+  PRECMD="$AOMP/bin/gpurun -s"
+fi
+$PRECMD ./milestone-3-babel -n 20               # 805.3 MB
 save_status
-./milestone-3-babel -n 20 -s 100000000  # 2400.0 MB
+set +x
```

Comment:
remove 'set +x' as it will turn off the above 'set -x' is there to show the exact commands used in the test log

Reply:
that is deliberate, i only wanted that much output
set -x
PRECMD=""
if [ -f $AOMP/bin/gpurun  ]; then
  PRECMD="$AOMP/bin/gpurun -s"
fi
$PRECMD ./milestone-3-babel -n 20               # 805.3 MB
save_status
set +x

Reply:
In the milestone-3-babel/doit.sh, the first 'set -x' was already there to make all of the commands show up in the log.

Reply:
The doit.sh script was originally turning on tracing for the entire script ...which is captured in the '<testname>.perf' log (which results are extracted from later by the CI).